examples/surface_defect_detection/README.md

```diff
@@ -0,0 +1,136 @@
+Download [dataset](https://github.com/abin24/Magnetic-tile-defect-datasets.) and the [label file](/examples/surface_defect_detection/data/2.txt) to `/data` of edge2.
+
+### Prepare Script
+
+Download the script [aggregate.py](/examples/surface_defect_detection/aggregation_worker/aggregate.py) to the `/code` of cloud node.
+
+Download the script [training_worker](/examples/surface_defect_detection/training_worker/train.py) to the `/code` of edge1 and edge2.
+
+
+### Create Federated Learning Job 
+
+#### Create Dataset
+
+```
+# create dataset for edge1
+kubectl create -f - <<EOF
+apiVersion: neptune.io/v1alpha1
+kind: Dataset
+metadata:
+  name: "edge1-surface-defect-detection-dataset"
+spec:
+  url: "/data/1.txt"
+  format: "txt"
+EOF
+
+# create dataset for edge2
+kubectl create -f - <<EOF
+apiVersion: neptune.io/v1alpha1
+kind: Dataset
+metadata:
+  name: "edge2-surface-defect-detection-dataset"
+spec:
+  url: "/data/2.txt"
+  format: "txt"
+EOF
+```
+
+#### Create Model
+
+```
+kubectl create -f - <<EOF
+apiVersion: neptune.io/v1alpha1
+kind: Model
+metadata:
+  name: "surface-defect-detection-model"
+spec:
+  url: "/model"
+  format: "ckpt"
+EOF
+```
+
+#### Start Federated Learning Job
+
+```
+kubectl create -f - <<EOF
+apiVersion: neptune.io/v1alpha1
+kind: FederatedLearningJob
+metadata:
+  name: surface-defect-detection
+spec:
+  aggregationWorker:
+    model:
+      name: "surface-defect-detection-model"
+    nodeName: "cloud0"
+    workerSpec:
+      scriptDir: "/code"
+      scriptBootFile: "aggregate.py"
+      frameworkType: "tensorflow"
+      frameworkVersion: "2.3"
+      parameters:
+        - key: "exit_round"
+          value: "3"
+  trainingWorkers:
+    - nodeName: "edge1"
+      dataset:
+        name: "edge-1-surface-defect-detection-dataset"
+      workerSpec:
+        scriptDir: "/code"
+        scriptBootFile: "train.py"
+        frameworkType: "tensorflow"
+        frameworkVersion: "2.3"
+        parameters:
+          - key: "batch_size"
+            value: "32"
+          - key: "learning_rate"
+            value: "0.001"
+          - key: "epochs"
+            value: "1"
+    - nodeName: "edge2"
+      dataset:
+          name: "edge-2-surface-defect-detection-dataset"
+      workerSpec:
+        scriptDir: "/code"
+        scriptBootFile: "train.py"
+        frameworkType: "tensorflow"
+        frameworkVersion: "2.3"
+        parameters:
+          - key: "batch_size"
+            value: "32"
+          - key: "learning_rate"
+            value: "0.001"
+          - key: "epochs"
+            value: "1"
+EOF
+```
+
+### Check Federated Learning Status
+
+```
+kubectl get federatedlearningjob surface-defect-detection
+```
+
+### Check Federated Learning Train Result
+After the job completed, you will find the model generated on the path `/home/data/model` in edge1 and edge2.
```

Comment:
`/home/data/model`? It should be `/model` at `cloud0`.

Reply:
The cloud worker do not save the model result. Only the edge worker will save. It should be `/model` in edge1 and edge2